contribs/gmf/src/services/treemanager.js

```diff
@@ -232,6 +274,9 @@ gmf.TreeManager.prototype.addGroupByLayerName = function(layerName, opt_add, opt
       var groupAdded = this.addFirstLevelGroups([group], opt_add, opt_silent);
       this.$timeout_(function() {
         var treeCtrl = this.getTreeCtrlByNodeId(group.id);
+        if (!treeCtrl) {
```

Comment:
(console.warn added . Perhaps can we do better, I've added this `if` cause tests was in error without that. But that's not the purpose of this PR)